specs.json

```
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
```

We're going to need that to point to the Bikeshed version, as our parsing tools cannot parse this format.

```suggestion
    "url": "https://webassembly.github.io/gc/core/bikeshed/",
```